content/release-10-17-0/announcements-10-17-0.md

```diff
@@ -204,3 +204,11 @@ instead of
 As of Cumulocity IoT 10.16, Machine Learning Engine and Machine Learning Workbench are deprecated, and we will end further development of these components. After Cumulocity IoT 10.17, both will be removed from the platform. If you have existing (production) projects which rely on MLE and/or MLW applications, then Software AG is committed to work together with you to find an appropriate transition path. If we have not yet been in contact for this matter, feel free to contact us at [c8y_mlw_mle_sunsetting@softwareag.com](mailto:c8y_mlw_mle_sunsetting@softwareag.com).
 
 For details, see the announcement in the release notes for [release 10.16](/release-10-16-0/announcements-10-16-0).
+
+### Digital Twin Manager
+
+#### Planned
+
+##### Deprecation of Assets move feature
```

Comment:
```suggestion
##### Deprecation of asset navigation
```

Reply:
@spau6 In the text you say that the asset navigation is deprecated, I wanted to make it consistent :-). If we call it asset move feature here we should call it the same in the text itself.

Reply:
Rephrasing this as 'Deprecation of asset navigation using Left navigation' in both header and text would be a better fit as we have replaced asset navigation using left nav with navigation using asset tree and subassets.

Reply:
How about saying "Deprecation of former asset navigation" in the header and "Deprecation of asset navigation using left navigation" in the text?